uiowa/uiowa
# UIHC Branding Update

Resolves https://github.com/uiowa/uiowa/issues/7403

# Test

- code review
- `ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli admin/appearance/settings/uids_base`
- See and change to the UIHC Healthcare option. Notice that the CCOM option is no longer present.
- Navigate to the front and see the UIHC Healthcare logo in the header and footer. Centered and seemingly the same width as the block IOWA logo on sandbox.prod.
- Note that the favicon is a yellow background icon with all of the variations available in the `<head>`.
- See that the SVG's fallback png is linked and available. (https://sandbox.uiowa.ddev.site/themes/custom/uids_base/assets/images/uihc.png)
